docs/_templates/index.html

```diff
@@ -33,6 +29,10 @@
     {"name": "Olympia", "state": "WA"}
   ]
 }</textarea>
+          <div class="left-inner-addon">
```

Comment:
This is admittedly a personal preference, but I prefer the expression to be on top.  It's the first thing a reader will see (which is what we want), and it's also consistent with the alternate format used in the examples pages with the side by side column format.